.env.example

```diff
@@ -8,3 +8,6 @@ UPRIGHT_API_TOKEN=
 SLACK_TOKEN=
 # From Slack channel preferences -> About -> Channel ID (Scroll to the bottom if not immediately visible)
 SLACK_CHANNEL=
+SLACK_ERROR_CHANNEL=
+
```

Comment:
Should we also add the login creds needed for the internal API?  :)

Reply:
Yes, feel free to update the file as new environment variables are introduced